src/config/wmodules-gcp.c

```diff
@@ -126,7 +126,7 @@ int wm_gcp_pubsub_read(xml_node **nodes, wmodule *module) {
             } else {
```

Comment:
There are still many `mwarn` in this file, have you checked them if they are correct?

Reply:
I check only those associated with the configurations that were in the table.

Reply:
Done